### PR TITLE
Layout: left panel with ecoverses

### PR DESCRIPTION
**Pull request by hero101**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/cherrytwist/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/cherrytwist/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/cherrytwist/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

**Review by techsmyth — APPROVED** [review]

File: src/components/layout/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  const styles = useStyles();
+
+  return (
+    <div id="sidebar-container" className={styles.sidebarContainer}>
+      <div id="sidebar-static-top" className={styles.sidebarStatic}>
+        <SidebarItem icon={HouseIcon} tooltip="Home" to="/" />
+      </div>
+      <div id="sidebar-dynamic" className={styles.sidebarDynamic}>
+        {ecoverses.map(x => (
+          <SidebarItemEcoverse ecoverse={x} />
```
Comment (atstojanov) [inline src/components/layout/Sidebar/Sidebar.tsx:62]: `Each child in a list should have a unique "key" prop.`
Something like:
```
{ecoverses.map((x,i) => (
          <SidebarItemEcoverse key={`ecoverse-${i}`} ecoverse={x} />
        ))}
```
https://reactjs.org/docs/lists-and-keys.html#keys

File: src/components/layout/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    marginTop: theme.shape.spacing(1),
+    marginBottom: theme.shape.spacing(1),
+  },
+}));
+
+const Sidebar: FC<SidebarProps> = ({ isUserAuth, ecoverses }) => {
+  const styles = useStyles();
+
+  return (
+    <div id="sidebar-container" className={styles.sidebarContainer}>
```
Comment (atstojanov) [inline src/components/layout/Sidebar/Sidebar.tsx:56]: Any purpose for the ids?

File: src/components/layout/Sidebar/SidebarItem.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+interface SidebarItemProps {
+  icon: FC<React.SVGProps<SVGSVGElement>>;
+  to: string;
+  tooltip: string;
+  disabled?: boolean;
+}
+
+const SidebarItem: FC<SidebarItemProps> = ({ icon, to, tooltip, disabled }) => {
+  return (
+    <div id="sidebar-item">
```
Comment (atstojanov) [inline src/components/layout/Sidebar/SidebarItem.tsx:16]: Setting static ids on items whish will be repeated it is not good idea. Some browsers can complain about this.

**Review by atstojanov — CHANGES_REQUESTED** [review]

**Review by atstojanov — APPROVED** [review]

File: src/components/layout/Sidebar/Sidebar.tsx (hunk 2 above)
Comment (hero101) [reply src/components/layout/Sidebar/Sidebar.tsx]: To identify them easier.

**Pull request merged**